azure/notebooks/remote-explanation/azure-gpu-shap.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>Can we run any part of this demo on multi-GPU? If not, we can just recommend single GPU VMs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
We cannot run it on multi-GPU, I'll use the VM 6s VM